build/packages-template/bbb-etherpad/opts-bionic.sh

```diff
@@ -1,5 +1,3 @@
 . ./opts-global.sh
 
-OPTS="$OPTS -d yq,tidy -t deb --deb-user etherpad --deb-group etherpad --deb-use-file-permissions"
-# OPTS="$OPTS -d yq,redis-server,libreoffice,tidy -t deb --deb-user etherpad --deb-group etherpad --deb-use-file-permissions"
-
+OPTS="$OPTS -d yq -t deb --deb-user etherpad --deb-group etherpad --deb-use-file-permissions"
```

Comment:
```suggestion
OPTS="$OPTS -d yq -t deb"
```

https://github.com/bigbluebutton/bigbluebutton/pull/14110/files#diff-ff8933b6c64dac81761e0a0129eabd52f1343e6a87364c30fb9cb4880684dedfR3

Reply:
Actually this has not made its way to 'develop' yet. nvm

Reply:
And I realized later on Etherpad does not use `yq` either. So we might be able to remove that one too.